easybuild/easyconfigs/j/Julia/Julia-1.0.0-foss-2018b.eb

```diff
@@ -0,0 +1,81 @@
+# Author: Jiri Furst <jiri.furst@gmail.com>, based on older eb by Jack Perdue <j-perdue@tamu.edu>
+
+easyblock = 'ConfigureMake'
+
+name = 'Julia'
+version = '1.0.0'
+
+homepage = 'http://julialang.org/downloads/'
+description = """
+ Julia is a high-level, high-performance dynamic programming language for
+ technical computing, with syntax that is familiar to users of other technical
+ computing environments. It provides a sophisticated compiler, distributed
+ parallel execution, numerical accuracy, and an extensive mathematical function
+ library.
+"""
+
+source_urls = ['https://github.com/JuliaLang/%(namelower)s/archive/v%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['aef997353def723862d76c2a4576bb330501bff20f87d44ad7eca3dca1890037']
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('M4', '1.4.18'),  
```

Comment:
@furstj failing style check is because of trailing whitespace on this line...

Reply:
Thanks, I will clean it up in a moment.